### ISSUE TITLE
[BUG] Unable to perfrom Room Setup

### ISSUE THREAD

**Issue by Nucleus1242** [ValveSoftware/SteamVR-for-Linux#648]
**Describe the bug**
I need to run the room setup, but I can't perform it.

**To Reproduce**

When SteamVR Home enabled:
1. Start SteamVR
1.1 See Windows (Img3 Img4 Img5) pop up
1.2. See SteamVR Home starting up in Headset
2. See Error (Img1)

Or

1. Start SteamVR
1.1 See Windows (Img3 Img4 Img5) pop up
2. Dismiss Error
3. Click on Perform Room Setup (Img2)
4. See Error (Img1)

With SteamVR Home disabled:
1. Start SteamVR
1.1 See Windows (Img3 Img4 Img5) pop up
1.2 See Room Setup starting up in Headset
2. There is no Room Setup window, even after waiting for several minutes

**Expected behavior**
I expect to see the window for the Room Setup, and being able to start it.

**System Information:**
- Distribution: Arch
- SteamVR version: beta 2.1.7
- Steam client version: Wed, Nov 22 6:54 PM UTC -08:00
- Opted into Steam client beta?: Yes
- Graphics driver version:  Mesa 23.2.1-arch1.2
- Gist for SteamVR System Information: https://gist.github.com/Nucleus1242/e7133bbb235fd0fa6e5f9d968c5a10e3

**Screenshots**
Img1
![image](https://github.com/ValveSoftware/SteamVR-for-Linux/assets/72988152/28c29848-626d-4894-bd00-7ce8c414d19a)
Img2
![image](https://github.com/ValveSoftware/SteamVR-for-Linux/assets/72988152/d67b0c0c-9315-4783-8271-ff6347e5cc7a)
Img3
![image](https://github.com/ValveSoftware/SteamVR-for-Linux/assets/72988152/2313d304-4242-4326-bc86-7bb9113f2377)
Img4
![image](https://github.com/ValveSoftware/SteamVR-for-Linux/assets/72988152/ca471976-f6e6-42e1-ac3b-4f29bb16c76c)
Img5
![image](https://github.com/ValveSoftware/SteamVR-for-Linux/assets/72988152/4d7865e4-596b-490e-98d0-b812ec4e5cc7)

**Additional context**
SteamVR was already running before, because I had a Room configuration. But I deleted it (there were some other reasons).
Those empty windows were already popping up for some versions but it didn't keep me from using SteamVR before.

**Note:** Commenters who are also experiencing this issue are encouraged to include the "System Information" section in their replies.

**Comment by CarbonNotes** [comment]
System Information (please complete the following information):

    Distribution: Debian Testing
    SteamVR version: 2.1.7 / 2.1.8
    Steam client version: 1700867040
    Opted into Steam client beta?: Yes
    Graphics driver version: Mesa 23.2.1-1:
    Gist for SteamVR System Information: https://gist.github.com/CarbonNotes/b53f5e64040d005d0aef6c2de59fdfaf


I was able to reproduce this with both my desktop (Debian Testing, as reported in initial issue description) with an Index as well as SteamOS 3.5.7 and an HTC Vive. In both cases, rolling back to 1.27.5 allowed room setup to complete, and moving forwards to 2.1.7 / 2.1.8 retained the room setup configured in 1.27.5.

**Comment by Nucleus1242** [comment]
I then also tried it in version 1.27.5 but unfortunately it didn't work. The Room Setup pops up for a second on VR startup with VR home disabled. After that it won't open by clicking the Start Room Setup button.
Corresponding SteamVR SysInfo: [https://gist.github.com/Nucleus1242/eb803732e94ee35f1a46c79d749636e1](url)

**Comment by Hicks79** [comment]
Hi @Nucleus1242 
same here
https://github.com/ValveSoftware/SteamVR-for-Linux/issues/646

**Comment by 2l47** [comment]
I’m unable to run room setup on the default branch, temp_v1.27.5 branch, or beta branch.

When I select room setup from the SteamVR menu, a window frame renders for a moment before disappearing. Subsequent attempts to launch it fail until SteamVR is restarted. Manually running the steamvr_room_setup binary ends in a segmentation fault.

**Comment by Patola** [comment]
Yeah, I can't run room setup too, archlinux, mesa 24.2.4, Radeon RX 7900 XTX, AMD 7800X3D, with Valve Index with knuckles.

**Comment by Patola** [comment]
getting back to temp_v1.27.5, running the room setup, then getting back to the normal version of steamvr allowed me to recalibrate.
